intelmq/bots/collectors/mail/collector_mail_url.py

```diff
@@ -90,7 +95,12 @@ def process(self):
                         # Only mark read if message relevant to this instance,
                         # so other instances watching this mailbox will still
                         # check it.
-                        mailbox.mark_seen(uid)
+                        try:
+                            mailbox.mark_seen(uid)
+                        except imaplib.abort:
+                            # Disconnect, see https://github.com/certtools/intelmq/issues/852
+                            mailbox = self.connect_mailbox()
```

Comment:
what happens in an exception inside the except block?

Reply:
The generic error handling of the Bot class handles this, like before and everywhere else too.

Reply:
OK